### PR TITLE
Adds font-metrics-overrides file

### PR DESCRIPTION
**Pull request by kennethormandy**
Adds font-metrics-overrides file, based on the included links and the links in #5660.

Things you might want to change:

- The filename could also be `font-metrics-override-desc` to match https://drafts.csswg.org/css-fonts-4/#font-metrics-override-desc, or be prefixed with `fontface` instead of just font? I’ve included the relevant keywords regardless—I’m not clear on the filename conventions
- I wasn’t clear on whether this being behind a feature flag meant `"86": "n d #1",` should actually be `a`lmost instead of `n`o

### CODE REVIEW

File: features-json/font-metrics-overrides.json [hunk 1]
```diff
@@ -0,0 +1,419 @@
...
+  "description": "Default font metrics overriding: the ascent-override, descent-override and line-gap-override descriptors",
+  "spec": "https://drafts.csswg.org/css-fonts-4/#font-metrics-override-desc",
+  "status": "unoff",
+  "links": [
+    {
+      "url": "https://docs.google.com/document/d/1PW-5ML5hOZw7GczOargelPo6_8Zkuk2DXtgfOtJ59Eo/edit",
+      "title": "New @font-face descriptors for overriding font metrics"
+    },
+    {
+      "url": "https://www.chromestatus.com/feature/5651198621253632",
```
Comment (Fyrd) [inline features-json/font-metrics-overrides.json:12]: This link will automatically be added thanks to the "chrome_id" value, so it can be removed here.

**Comment by Fyrd** [comment]
File name looks good and `n d #1` is correct, since there's no support by default. 👍

**Comment by kennethormandy** [comment]
Well I guess I made the case for the CI, so thanks for merging that ha. Let me know if you want me to squash these down to one commit.

**Comment by Fyrd** [comment]
Thanks! Already squashed.

**Pull request merged**